### ISSUE TITLE
Blank Screen

### ISSUE THREAD

**Issue by ounverdi** [zoom/zoomapps-sample-js#1]
Hi,

Thanks for that great repo which is very useful for app developers.

I have followed the steps given and created my first app.

App can be installed and opened on Zoom-Client but shows blank screen instead of content of site.

I dont know is there any way to debug/inspect/checking console logs inside Zoom-client.

<img width="1038" alt="Screen Shot 2022-06-29 at 14 44 22" src="https://user-images.githubusercontent.com/96845668/176428764-7d123d84-0df5-4200-92a5-6b5ea90877bd.png">

**Comment by guatedude2** [comment]
I have the same problem. I ran Zoom via command line and the logs show this:

```
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>RDQK04XKYV5B5NTB</RequestId><HostId>mlcTxCn4mznj1Pq+XXXXXX=</HostId></Error><?xml version="1.0" encoding="UTF-8"?>
```

**Comment by SafeeSaif** [comment]
To enable Inspect Element in the Zoom Client, follow these directions:

Mac:
run the following command in your terminal:
`defaults write ZoomChat webview.context.menu true`

Windows:
Open the file %appdata%/Zoom/data/Zoom.us.ini
Add this to the [ZoomChat] section
`[ZoomChat]
webview.context.menu=true`

**Comment by ounverdi** [comment]
Thanks a lot!

It seems that never loads page

<img width="1080" alt="Screen Shot 2022-06-30 at 01 39 06" src="https://user-images.githubusercontent.com/96845668/176557730-d30b760f-4b85-4d48-8eee-b605cdfe3e80.png">

**Comment by guatedude2** [comment]
Great I found the issue looks like it's a security header that's missing in the sample app

**Comment by ounverdi** [comment]
Yes I agree. I have added required headers and now it is working.

**Comment by guatedude2** [comment]
Hey @ounverdi can you add a PR back to the sample?

**Comment by ounverdi** [comment]
Problem is not related with source code, it is related with ngrok's trial version which is not delivering headers as expected.

I have added the following headers to my server.

>    Content-Security-Policy
>     Strict-Transport-Security
>     X-Content-Type-Options
>     Referrer-Policy

**Comment by guatedude2** [comment]
Oh great. Thanks I'll try that

**Comment by guatedude2** [comment]
I wasn't able to fix this issue, tried different ways but looks like it can't be done with ngrok trial

**Comment by ounverdi** [comment]
Great!

Yes It should be added

**Comment by ArpitShuklaTI** [comment]
```
Problem is not related with source code, it is related with ngrok's trial version which is not delivering headers as expected.

I have added the following headers to my server.

Content-Security-Policy
Strict-Transport-Security
X-Content-Type-Options
Referrer-Policy
```
@ounverdi Can you please tell how did you add headers to your server?

**Comment by ounverdi** [comment]
Actually, our react app is running on GCloud and  have added following blocks to my yml file.

`  http_headers: 
    Content-Security-Policy: "default-src * self blob: data: gap:; style-src * self 'unsafe-inline' blob: data: gap:; script-src * 'self' 'unsafe-eval' 'unsafe-inline' blob: data: gap:; object-src * 'self' blob: data: gap:; img-src * self 'unsafe-inline' blob: data: gap:; connect-src self * 'unsafe-inline' blob: data: gap:; frame-src * self blob: data: gap:;"
    Strict-Transport-Security: max-age=31536000;includeSubDomains
    X-Content-Type-Options: "nosniff"
    Referrer-Policy: no-referrer`

**Comment by ounverdi** [comment]
Are you using trial version of ngrok?

**Comment by ounverdi** [comment]
Unfortunately it ignores given headers due to trial limitations.

**Comment by ArpitShuklaTI** [comment]
Oh. So is there no way to make it work with trial version?

**Comment by ounverdi** [comment]
Yes, no way.

**Comment by ArpitShuklaTI** [comment]
Okay. Thanks for the quick response.

**Comment by geneluong-cs** [comment]
You can actually achieve this with ngrok's traffic policy
https://ngrok.com/docs/traffic-policy/actions/add-headers/

```yaml
on_http_response:
  - actions:
      - type: add-headers
        config:
          headers:
            X-Content-Type-Options: "nosniff"
```

Then launch the server with the traffic policy

`$ ngrok http 80 --traffic-policy-file policy.yml`
